### PR TITLE
chore: add round and remove shadow for menu item

### PR DESCRIPTION
**Pull request by FeiWang1119**
add round and remove shadow for menu item

Log:

### CODE REVIEW

**Comment by FeiWang1119** [comment]
Uploading 1.mp4…

**Comment by FeiWang1119** [comment]
https://github.com/linuxdeepin/qt5integration/assets/102005754/f985da2a-6e76-433a-8b13-7006730b16da

**Comment by 18202781743** [comment]
加圆角和移除阴影的代码分开提交吧,

File: styleplugins/chameleon/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +16,6 @@ dtk_add_plugin(
         common.h
         dstyleanimation.h
     RESOURCES
```
Comment (18202781743) [inline styleplugins/chameleon/CMakeLists.txt:18]: RESOURCES关键字也可以删掉,

File: styleplugins/chameleon/chameleonstyle.cpp [hunk 2]
```diff
@@ -2944,17 +2883,23 @@ ChameleonMovementAnimation *ChameleonStyle::drawMenuItemBackground(const QStyleO
             if (selected)
                 animation->setTargetRect(option->rect);
         }
-
+        const int round = 6, margin = 6;
```
Comment (18202781743) [inline styleplugins/chameleon/chameleonstyle.cpp:2886]: margin 应该可以用 PM_MenuHMargin, 圆角看要不要也从那里拿,

File: styleplugins/chameleon/chameleonstyle.cpp [hunk 3]
```diff
@@ -2944,17 +2883,23 @@ ChameleonMovementAnimation *ChameleonStyle::drawMenuItemBackground(const QStyleO
...
+            painter->setPen(Qt::NoPen);
+            QRect rect = animation->currentRect().adjusted(margin, 0, - margin, 0);
+            painter->drawRoundedRect(rect, round, round);
             painter->setOpacity(opacity);
 
             return animation;
         } else if (selected) {
-            painter->fillRect(option->rect, color);
+            painter->setBrush(color);
+            painter->setPen(Qt::NoPen);
```
Comment (18202781743) [inline styleplugins/chameleon/chameleonstyle.cpp:2900]: pen 要不要restore下,不然就丢了,

File: styleplugins/chameleon/CMakeLists.txt (hunk 1 above)
Comment (FeiWang1119) [reply styleplugins/chameleon/CMakeLists.txt:18]: done

File: styleplugins/chameleon/chameleonstyle.cpp (hunk 2 above)
Comment (FeiWang1119) [reply styleplugins/chameleon/chameleonstyle.cpp:2886]: done

File: styleplugins/chameleon/chameleonstyle.cpp (hunk 3 above)
Comment (FeiWang1119) [reply styleplugins/chameleon/chameleonstyle.cpp:2902]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**